Userland/Libraries/LibWeb/WebAudio/BiquadFilterNode.cpp

```diff
@@ -85,6 +85,19 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<BiquadFilterNode>> BiquadFilterNode::constr
     // MUST initialize the AudioNode this, with context and options as arguments.
 
     auto node = realm.vm().heap().allocate<BiquadFilterNode>(realm, realm, context, options);
+
+    // Default options for channel count and interpretation
+    // https://webaudio.github.io/web-audio-api/#BiquadFilterNode
+    AudioNodeDefaultOptions default_options;
+    default_options.channel_count_mode = Bindings::ChannelCountMode::Max;
+    default_options.channel_interpretation = Bindings::ChannelInterpretation::Speakers;
+    default_options.channel_count = 2;
+    // FIXME: Set tail-time to yes
+
+    TRY(node->initialize_audio_node_options(options, default_options));
+
+    TRY(node->set_type(options.type));
```

Comment:
It would be a bit better to do this in the constructor, especially given `set_type` should not throw as I understand from reading the spec